backend/app.js

```diff
@@ -26,6 +28,15 @@ app.use((req, res, next) => {
   next();
 });
 
+var corsOption = {
+  origin: true,
+  methods: 'GET,HEAD,PUT,PATCH,POST,DELETE',
+  credentials: true,
+  exposedHeaders: ['x-auth-token']
+};
+app.use(cors(corsOption));

```

Comment:
Many of these options are already specified above, don't need to repeat them. You could remove either ones